tgui/packages/tgui/interfaces/GPS.tsx

```diff
@@ -0,0 +1,214 @@
+                    step={1}
+                    onChange={(x_val: number) => act('set_x', { x: x_val })}
+                  />
+                  y:{' '}
+                  <NumberInput
+                    value={track_y}
+                    width={'3'}
+                    minValue={1}
+                    maxValue={300}
+                    step={1}
+                    onChange={(y_val: number) => act('set_y', { y: y_val })}
+                  />
+                  <Button
+                    mt={0.5}
+                    onClick={() =>
+                      act('track_coords', { x: track_x, y: track_y })
+                    }
+                  >
+                    Track
+                  </Button>
+                </LabeledList.Item>
+              </LabeledList>
+            </Stack.Item>
+            <Stack.Item>
+              <LabeledList>
+                <LabeledList.Item label="Trackable">
+                  <Button.Checkbox
+                    onClick={() => act('toggle_trackable')}
+                    selected={trackable}
+                    checked={trackable}
+                  >
+                    {trackable ? 'Yes' : 'No'}
+                  </Button.Checkbox>
+                </LabeledList.Item>
+                <LabeledList.Item label="Send distress">
+                  <Button.Checkbox
+                    onClick={() => act('toggle_distress')}
+                    selected={distress}
+                    checked={distress}
+                  >
+                    {distress ? 'Yes' : 'No'}
+                  </Button.Checkbox>
+                </LabeledList.Item>
+                <LabeledList.Item label="Tracking">
+                  <Button onClick={() => act('track_gps')}>
+                    {tracking ? tracking : 'None'}
+                  </Button>
+                </LabeledList.Item>
+              </LabeledList>
+            </Stack.Item>
+          </Stack>
+        </Section>
+        <Section title="Tracking">
+          <Collapsible title="GPS Devices">
+            {!!gps_info.length && (
+              <Stack vertical>
+                <TrackableList gps_info={gps_info} />
+              </Stack>
+            )}
+          </Collapsible>
+          <Collapsible title="Implants">
+            {!!imp_info.length && (
+              <Stack vertical>
+                <TrackableList gps_info={imp_info} />
+              </Stack>
+            )}
+          </Collapsible>
+          <Collapsible title="Warp Beacons">
+            {!!warp_info.length && (
+              <Stack vertical>
+                <TrackableList gps_info={warp_info} />
+              </Stack>
+            )}
+          </Collapsible>
+        </Section>
+      </Window.Content>
+    </Window>
+  );
+};
+
+let distress_red = false;
+const TrackableList = (props) => {
+  const { act } = useBackend();
+  const { gps_info } = props;
+  const nodes: JSX.Element[] = [];
+
+  distress_red = !distress_red;
```

Comment:
Ack, no.

If you want this flashing, use CSS animations or something.

Reply:
Where can I find CSS animations or examples?

Reply:
We don't use them extensively (for good reason), but the ChemHeater interface does an oscillating thing.

https://github.com/goonstation/goonstation/blob/23bbebf599a2b041bc8f4b180cb2ed2a9808f57b/tgui/packages/tgui/styles/interfaces/ChemHeater.scss#L51